tests/testsuite/package.rs

```diff
@@ -1870,15 +1870,11 @@ fn long_file_names() {
         test_path.mkdir_p();
         let test_path = test_path.join(long_name);
         if let Err(e) = File::create(&test_path) {
-            use std::io::Write;
-            writeln!(
-                std::io::stderr(),
+            eprintln!(
                 "\nSkipping long_file_names test, this OS or filesystem does not \
                 appear to support long file paths: {:?}\n{:?}",
-                e,
-                test_path
-            )
-            .unwrap();
+                e, test_path
+            );
```

Comment:
This was specifically written to write to stderr to avoid libtest from capturing the output (so that it is always displayed, even without `--nocapture`).  Can you change it back?  (And maybe add a comment explaining that, since it isn't obvious.)